proposals/interface-types/working-notes/choices/webidl.md

```diff
@@ -0,0 +1,270 @@
+# Interface Types and WebIDL
+
+## Preamble
+This note is part of a series that explain some of the major design choices in
+the Interface Types specification.
+
+## Introduction
+
+A key motivation for Interface Types is the semantic gap between the
+expressivitiy of core WebAssembly and the needs of API designers and application
+authors. Part of this gap is represented by the difference between the type
+system of core WebAssembly and the needs of a typical programmer.
+
+As an example of this, core WebAssembly has no intrinsic type corresponding to
+`string`; even though this is arguably the most important type in many if not
```

Comment:
nit: I'd avoid saying string "is arguably the most important" type, but rather "is present in most interfaces".  ("most important" suggests "stringly typed" programming, which I personally think is an antipattern ;)

Reply:
This is an opinion I agree. It is based on my xx-ty years of experience in developing systems and applications. Nothing to do with types per se.
I will reword; but my opinion stands.

Reply:
reworded to 'extremely popular type'